Avoid useless makedependency update-desktop-database since it isn't
needed when DESTDIR is used.

--- a/meson.build
+++ b/meson.build
@@ -110,7 +110,3 @@
     'scripts/xinput_calibrator.xpm',
     install_dir: get_option('datadir') / 'pixmaps'
 )
-
-gnome.post_install(
-    update_desktop_database: true,
-)
